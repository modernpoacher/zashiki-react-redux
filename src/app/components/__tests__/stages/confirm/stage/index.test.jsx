import React from 'react'

import {
  toSnapshot
} from 'react-component-snapshot'

import '@testing-library/jest-dom'

import {
  render
} from '@testing-library/react'

import Stage from '#zashiki-react-redux/app/components/stages/confirm/stage'

const MOCK_DEFINITION = {
  meta: {
    uri: '#/'
  },
  elements: {
    title: 'MOCK TITLE',
    description: 'MOCK DESCRIPTION',
    field: {
      id: 'MOCK ID'
    }
  }
}

const MOCK_RESPONSE = {
  '#/': 'MOCK VALUE'
}

describe('#zashiki-react-redux/app/components/stages/confirm/stage', () => {
  describe('Always', () => {
    it('renders', () => {
      expect(toSnapshot(render(
        <Stage
          definition={MOCK_DEFINITION}
          response={MOCK_RESPONSE}
          onChange={jest.fn()}
          onSubmit={jest.fn()}
        />
      )))
        .toMatchSnapshot()
    })
  })
})
