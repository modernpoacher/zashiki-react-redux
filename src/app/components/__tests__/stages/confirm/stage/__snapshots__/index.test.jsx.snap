// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#zashiki-react-redux/app/components/stages/confirm/stage Always renders 1`] = `
<div
  className="confirm"
>
  <h2>
    Confirmation
  </h2>
</div>
`;
